Add monitor docs to docsv2 project (#9949)
netdata/netdata: docs/monitor/view-active-alarms.md

<!--
title: "View active health alarms"

> description: "View active alarms and their rich data to discover and resolved anomalies and performance issue across your infrastructure."

Reviewer: ```suggestion
description: "View active alarms and their rich data to discover and resolve anomalies and performance issues across your infrastructure."
```

what that chart's current value is, the alarm that triggered it, and when the alarm status first began.

You can use the input field in the Alarms panel to filter active alarms. You can sort by the node's name, alarm, status,
chart that triggered the alarm, or the operating system. Read more about the [filtering
syntax](/docs/visualize/view-all-nodes.md#filter-and-group-your-infrastructure) to build valuable filters for your
infrastructure.

Click on the 3-dot icon (`⋮`) to view active alarm information or navigate directly to the offending chart in that
node's Cloud dashboard with the **Go to chart** button.

The active alarm information gives you in-depth information about the alarm that's been triggered. You can see the
alarm's configuration, how it calculates warning or critical alarms, and which configuration file you could edit on that
node if you want to tweak or disable the alarm to better suit your needs.

![Screenshot from 2020-09-17
17-21-29](https://user-images.githubusercontent.com/1153921/93541139-710eb580-f90a-11ea-809d-25afe1270108.png)

## View active alarms in the Netdata Agent

> Find the bell 🔔 icon in the top navigation bring up a modal that shows currently raised alarms, all running alarms, and

Reviewer: ```suggestion
Find the bell 🔔 icon in the top navigation to bring up a modal that shows currently raised alarms, all running alarms, and
```

the alarms log. Here is an example of a raised `system.cpu` alarm, followed by the full list and alarm log:

![Animated GIF of looking at raised alarms and the alarm
log](https://user-images.githubusercontent.com/1153921/80842482-8c289500-8bb6-11ea-9791-600cfdbe82ce.gif)

And a static screenshot of the raised CPU alarm: 

![Screenshot of a raised system CPU
alarm](https://user-images.githubusercontent.com/1153921/80842330-2dfbb200-8bb6-11ea-8147-3cd366eb0f37.png)

The alarm itself is named **system - cpu**, and its context is `system.cpu`. Beneath that is an auto-updating badge that

> shows the latest value the chart that triggered the alarm.

Reviewer: ```suggestion
shows the latest value of the chart that triggered the alarm.
```
